Developers/Writing_Good_Commits.rst

```diff
@@ -23,9 +23,9 @@ How to Write Good Commit Messages
 
 A commit message consists of 3 parts:
```

Comment:
also, try holding a shortlog limit of 50 chars as else github wraps commit messages badly and its unreadable. thanks!